projects/1.18/assets/botania/botania/lang/zh_cn.json

```
@@ -43,20 +44,20 @@
   "botaniamisc.dropTip1": "Ctrl+%s 扔出一整组",
   "botaniamisc.wandMode.bind": "绑定模式",
   "botaniamisc.wandMode.function": "工作模式",
-  "botaniamisc.brewOf": "%s的酿造",
-  "botaniamisc.needsCatalysts": "无论你多么努力地尝试将泰拉钢锭推送入信标之中，但就是什么事都没有发生。也许你用于仪式的水晶塔摆错了位置。",
+  "botaniamisc.brewOf": "%s精酿",
+  "botaniamisc.needsCatalysts": "无论你多么努力地尝试将泰拉钢锭送入信标之中，什么事都没有发生。也许你用于仪式的水晶塔摆错了位置。",
   "botaniamisc.peacefulNoob": "你尝试提供泰拉钢锭，但是被和平模式世界当中的和平共鸣所阻止。也许你该采取些措施。",
   "botaniamisc.notInfused": "未灌注",
-  "botaniamisc.getALife": "获得一条生命",
+  "botaniamisc.getALife": "你难道没有现实生活吗？",

```

原文 `Get a Life`，这是自创中文烂梗？

---

还有这个意思，但是和此处`现实生活`的翻译没有太大关系，Get a life 似乎并没有对 `现实` 的强调或对比

---

看代码是给泰拉粉碎者充魔力升级到Java最大整数的时候会加一句吐槽，意思是“你闲得没事吗”
```suggestion
  "botaniamisc.getALife": "你闲得没事吗？",
```
```suggestion
  "botaniamisc.getALife": "过你自己的生活",
```